tests/test_secrets.py

```diff
@@ -238,3 +243,80 @@ class Cfg(object):
         cfg = environ.to_config(Cfg, fake_environ)
 
         assert _SecretStr("foo") == cfg.pw
+
+
+class TestDirectorySecrets(object):
+    @pytest.fixture
```

Comment:
There's no need to keep this within the class. Just move it out to module scope.